File: VonexDealer.Vue/ClientApp/models/hardware.js
export class hardware {
    constructor({

        hardwareID = 0,
        hardwareOrderID = 0,
        quantity = 1,
        cost = 0.00,
        handsetID = 0,
        comment = null,
        dealerSupplied = false,
        lineTotal = 0.00,
        fromIPOrder = false,
    } = {}) {
        this.hardwareID = hardwareID;
        this.hardwareOrderID = hardwareOrderID;
        this.quantity = quantity;
        this.handsetID = handsetID;
        this.comment = comment;
        this.dealerSupplied = dealerSupplied;
        this.cost = cost;
        this.lineTotal = lineTotal;
        this.fromIPOrder = fromIPOrder
    }
  

}

export function CreateHardware(data) {
    return Object.freeze(new hardware(data))
}

File: VonexDealer.API/Models/Order/Hardware.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace VonexDealer.API.Models.Order
{
    public class Hardware
    {
        public Int64 HardwareID { get; set; }
        public Int64 HardwareOrderID { get; set; }
        public int Quantity { get; set; }
        public Int64 HandsetID { get; set; }
        public string Comment { get; set; }
        public bool DealerSupplied { get; set; }
        public double Cost { get; set; }
        public double LineTotal { get; set; }
        public bool FromIPOrder { get; set; }

    }
}
